### ISSUE TITLE
[Bug]: Calling get_all_assets returns a validation error

### ISSUE THREAD

**Issue by enicolasgomez** [alpacahq/alpaca-py#276]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

calling `TradingClient.get_all_assets(GetAssetsRequest(status=AssetStatus.ACTIVE))` results in a bunch of errors like  this:

```
validation errors for ParsingModel[List[alpaca.trading.models.Asset]]
__root__ -> 11105 -> exchange
  value is not a valid enumeration member; permitted: 'AMEX', 'ARCA', 'BATS', 'NYSE', 'NASDAQ', 'NYSEARCA', 'FTXU', 'CBSE', 'GNSS', 'ERSX', 'OTC' (type=type_error.enum; enum_values=[<AssetExchange.AMEX: 'AMEX'>, <AssetExchange.ARCA: 'ARCA'>, <AssetExchange.BATS: 'BATS'>, <AssetExchange.NYSE: 'NYSE'>, <AssetExchange.NASDAQ: 'NASDAQ'>, <AssetExchange.NYSEARCA: 'NYSEARCA'>, <AssetExchange.FTXU: 'FTXU'>, <AssetExchange.CBSE: 'CBSE'>, <AssetExchange.GNSS: 'GNSS'>, <AssetExchange.ERSX: 'ERSX'>, <AssetExchange.OTC: 'OTC'>])
```

### SDK Version I encountered this issue in

Latest

### Steps To Reproduce

```markdown
Just run the above line with a valid account and paper mode.
```

### Filled out the Steps to Reproduce section?

- [X] I have entered valid steps to reproduce my issue or have attached a minimally reproducible case in code that shows my issue happening; and understand that without this my issue will be flagged as invalid and closed after 30 days.

**Comment by dasaqui** [comment]
I can confirm the same behaviour with version 7 and latest version of the code, also for get_all_assets function

**Comment by jonkaa** [comment]
missing  `CRYPTO = "CRYPTO"` in `class AssetExchange(str, Enum):` in alpaca/trading/enums.py

**Comment by enicolasgomez** [comment]
I temporary fixed it by querying all US EQUITIES (seems the only parameter to be working) and filtering client side. This will deff produce big chunk of data being transfered as it's around 40K assets, my previous request was 10X less.

```
      search_params = GetAssetsRequest(asset_class=AssetClass.US_EQUITY)
      all_assets = self._trading_client.get_all_assets(search_params)
      all_assets = [a.symbol for a in all_assets if 
                    a.exchange in (AssetExchange.NYSE, AssetExchange.NASDAQ) 
                    and a.marginable 
                    and a.shortable 
                    and a.status is AssetStatus.ACTIVE]
```

**Comment by brettelliot** [comment]
Im seeing this error also when calling `get_all_positions`

`positions = trading_client.get_all_positions()
2023-04-26T13:09:45.920959+00:00 app[web.1]:   File "/usr/local/lib/python3.9/site-packages/alpaca/trading/client.py", line 240, in get_all_positions
2023-04-26T13:09:45.920959+00:00 app[web.1]:     return parse_obj_as(List[Position], response)
2023-04-26T13:09:45.920959+00:00 app[web.1]:   File "pydantic/tools.py", line 38, in pydantic.tools.parse_obj_as
2023-04-26T13:09:45.920959+00:00 app[web.1]:   File "pydantic/main.py", line 341, in pydantic.main.BaseModel.__init__
2023-04-26T13:09:45.920959+00:00 app[web.1]: pydantic.error_wrappers.ValidationError: 2 validation errors for ParsingModel[List[alpaca.trading.models.Position]]
2023-04-26T13:09:45.920959+00:00 app[web.1]: __root__ -> 0 -> exchange
2023-04-26T13:09:45.920959+00:00 app[web.1]:   value is not a valid enumeration member; permitted: 'AMEX', 'ARCA', 'BATS', 'NYSE', 'NASDAQ', 'NYSEARCA', 'FTXU', 'CBSE', 'GNSS', 'ERSX', 'OTC' (type=type_error.enum; enum_values=[<AssetExchange.AMEX: 'AMEX'>, <AssetExchange.ARCA: 'ARCA'>, <AssetExchange.BATS: 'BATS'>, <AssetExchange.NYSE: 'NYSE'>, <AssetExchange.NASDAQ: 'NASDAQ'>, <AssetExchange.NYSEARCA: 'NYSEARCA'>, <AssetExchange.FTXU: 'FTXU'>, <AssetExchange.CBSE: 'CBSE'>, <AssetExchange.GNSS: 'GNSS'>, <AssetExchange.ERSX: 'ERSX'>, <AssetExchange.OTC: 'OTC'>])`

**Comment by haxdds** [comment]
@bourba85 This fix is available in `v0.8.2`. Try upgrading with `pip install -U alpaca-py`
